src/app/components/navbar.tsx

```diff
@@ -0,0 +1,9 @@
+export default function Navbar() {
+  return (
+    <div className="flex mt-auto sticky bottom-0">
```

Comment:
let's use semantic html tags
```suggestion
    <nav className="flex mt-auto sticky bottom-0">
```

Reply:
already updated on https://github.com/RR-Ardhika/murojaah/pull/3/commits/03e94125dce8075c8df7a729973197ff0498dae9